product-service/src/functions/createProduct/handler.ts

```diff
@@ -0,0 +1,22 @@
+import { APIGatewayEvent } from 'aws-lambda';
+import { middyfy } from '@libs/lambda';
+import { Client } from 'pg';
+import dbConnection from '@libs/db-connection';
+
+const createProduct = async (event: APIGatewayEvent): Promise<void> => {
+  console.log(event);
+
+  const client = new Client(dbConnection);
+  await client.connect();
+  const { title, description, price } = event as any;
+
+  try {
+    await client.query(`INSERT INTO public.products (title, description, price) VALUES ('${title}', '${description}', ${price})`);
```

Comment:
It would be better if used parametrized query, like here. This kind of usage is vulnerable to SQL injections, which are very dangerous.
```suggestion
    await client.query(`INSERT INTO public.products (title, description, price) VALUES ($1, $2, $3)`, [title, description,price]);
```
https://node-postgres.com/features/queries#parameterized-query